listenerutil/parse.go

```diff
@@ -387,3 +413,142 @@ func ParseSingleIPTemplate(ipTmpl string) (string, error) {
 		return "", fmt.Errorf("multiple addresses found (%q), please configure one", out)
 	}
 }
+
+const strictTransportSecurity = "max-age=31536000; includeSubDomains"
```

Comment:
Can you name these with a bit more definition? e.g. `defaultStrictTransportSecurityHeader`, `defaultXContentTypeOptionsHeader", `defaultCacheControlHeader`, etc.